config/application.rb

```diff
@@ -20,5 +20,7 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
     ActiveSupport.escape_html_entities_in_json = false
+
+    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
I only changed this because Rails decided to start printing a ton of warnings that not specifying this setting was deprecated.